web/source/osk/visualKeyboard.ts

```diff
@@ -1201,7 +1208,7 @@ namespace com.keyman.osk {
       return null;
     }
 
-    show(host: OSKManager) {
+    updateState() {
```

Comment:
It never really was a "show" method... and not really even an "update layout" method.

It's also 100% internal, so the name-change is safe.